sycl/test-e2e/Regression/commandlist/Inputs/main.cpp

```diff
@@ -76,8 +76,7 @@ int main(int argc, char *argv[]) {
   std::vector<sycl::device> dlist;
   if (gpu_dev == 999) {
     try {
-      auto sel = sycl::gpu_selector();
-      sel_dev = sel.select_device();
+      sel_dev = sycl::device(sycl::gpu_selector_v);
```

Comment:
That doesn't do much given https://github.com/intel/llvm/blob/c5e4d783cb630d97049da3bce782a5b8836a0c47/sycl/test-e2e/Regression/commandlist/gpu.cpp#L5. I think we should either use implicit default selector in this file or adjust the other file to maybe use `%{run-unfiltered-devices}`.